### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by antonio-dropulic**
<del>Had a lot of rustfmt changes, not sure if it's miss-configured somehow.</del>

### CODE REVIEW

File: bin/millau/node/src/service.rs [hunk 1]
```diff
@@ -107,15 +109,15 @@ pub fn new_partial(
...
+		sc_service::new_full_parts::<Block, RuntimeApi, _>(
+			&config,
 			telemetry.as_ref().map(|(_, telemetry)| telemetry.handle()),
 			executor,
 		)?;
 	let client = Arc::new(client);
 
 	let telemetry = telemetry.map(|(worker, telemetry)| {
-		task_manager.spawn_handle().spawn("telemetry", worker.run());
+		task_manager.spawn_handle().spawn("telemetry", None, worker.run());
```
Comment (antonio-dropulic) [inline bin/millau/node/src/service.rs:120]: `SpawnTaskHandle` introduced groups. I've set all to default(None).

File: bin/millau/node/src/command.rs [hunk 2]
```diff
@@ -88,70 +88,74 @@ pub fn run() -> sc_cli::Result<()> {
...
 			runner
 				.sync_run(|config| cmd.run::<Block, RuntimeApi, service::ExecutorDispatch>(config))
-		},
+		}
 		None => {
 			let runner = cli.create_runner(&cli.run)?;
 			runner.run_node_until_exit(|config| async move {
 				match config.role {
-					Role::Light => service::new_light(config),
+					Role::Light => {
```
Comment (antonio-dropulic) [inline bin/millau/node/src/command.rs:151]: Should this be handled differently?

File: bin/millau/node/src/service.rs (hunk 1 above)
Comment (svyatonik) [reply bin/millau/node/src/service.rs:120]: Millau node is (mostly) a copy of https://github.com/paritytech/substrate/tree/master/bin/node-template . So if it is`None` there, then it should be fine

File: bin/millau/node/src/command.rs [hunk 3]
```diff
@@ -147,7 +147,10 @@ pub fn run() -> sc_cli::Result<()> {
...
```
Comment (svyatonik) [inline bin/millau/node/src/command.rs:150]: Let's just remove it as it is done in substrate node template: https://github.com/paritytech/substrate/blob/master/bin/node-template/node/src/command.rs#L113

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**